src/main/java/wolox/training/controllers/UserController.java

```diff
@@ -49,4 +50,14 @@ public User updateUser(@RequestBody User User, @PathVariable Long id) {
         return userRepository.save(User);
     }
 
+    @PostMapping("/{id}/books")
+    public void addBook(@RequestBody Book book, @PathVariable Long id) {
```

Comment:
You don't need to receive the full book, given you can receive the id and fetch it from the database.